extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -197,7 +197,7 @@ export class User implements IUserContext {
         // @DynamicProtoStub -- DO NOT add any code as this will be removed during packaging
     }
 
-    public update() {
+    public update(userId?: string) {
```

Comment:
please add the missing ';' after the bracket on line 178 (above) to avoid the Code scanning warning -- since we are touching this file.